src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -13019,6 +13019,105 @@ private static Vector512<T> IsNegative<T>(Vector512<T> vector)
             return Vector512.LessThan(vector, Vector512<T>.Zero);
         }
 
+        /// <summary>poly = C0 + C1*r^3 + C2*r^5 + C3*r^7 + C4*r^9</summary>
```

Comment:
Should this comment be C0*r ? It seems a1 is multiplied by r at the end?

Reply:
Agreed, the implementation seems to suggest that C0 is multiplied by `r`. Otherwise the expansion looks correct.